.github/workflows/pr-deploy.yaml

```diff
@@ -26,6 +26,7 @@ jobs:
       deployments: write
     uses: ./.github/workflows/netlify.yaml
     with:
+      artifact_run_id: ${{ github.event.workflow_run.id || github.run_id }}
```

Comment:
Is the `|| github.run_id` part necessary if this parameter also has that as its default?

Reply:
tbd. I don't know. Not sure if it would handover undefined or similar. Hence, rather a little redundant